contributors/design-proposals/api-machinery/dynamic-admission-control-configuration.md

```diff
@@ -282,7 +282,7 @@ See [Considered but REJECTED alternatives](#considered-but-rejected-alternatives
 2. #1 will allow parallel initializers as well.
 
 3. implement the fail closed initializers according to
-   [proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/admission_control_extension.md#initializers).
+   [proposal](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/admission_control_extension.md#initializers).
```

Comment:
same folder. this can be:
`[proposal](admission_control_extension.md#initializers)`